src/Gtk/Perspex.Gtk/GtkPlatform.cs

```diff
@@ -10,6 +10,18 @@
 using Perspex.Platform;
 using Perspex.Shared.PlatformSupport;
 
+namespace Perspex
+{
+	public static class GtkApplicationExtensions
+	{
+		public static AppT UseGtkSubsystem<AppT>(this AppT app) where AppT : Application
```

Comment:
Could do, i wanted to somehow denote between windowing/input and rendering. But in some cases they are also locked together. I will ditch the Subsystem suffix for now.